free_fleet_adapter/free_fleet_adapter/fleet_adapter.py

```diff
@@ -133,7 +133,10 @@ def main(argv=sys.argv):
         tf = compute_transforms(level, coords, node)
         fleet_config.add_robot_coordinates_transformation(level, tf)
 
+    # retrieves std::shared_ptr<EasyFullControl>
     fleet_handle = adapter.add_easy_fleet(fleet_config)
+    assert fleet_handle is not None, \
+    "An error was raised on attempt to create EasyFullControl Object"
```

Comment:
```suggestion
        "Failed to create EasyFullControl fleet, please verify that the fleet config is valid."
```
for linting, and making the error more informative